phobos/simple_deployment.md

```diff
@@ -0,0 +1,182 @@
+# Install rpc-ceph in the Phobos lab OnMetal 
+
+
+## Create Environment
+* 1 admin vm
+* 1 rsyslog vm
+* 5 boxes for storage nodes
+
+```bash
```

Comment:
I guess the network-id, sec-group and image id may change over time? I'm not sure we need to say how to get those, but may be worth pointing out those values arent necessarily going to work as a copy paste.

oh and to use your own key rather than Jay's key!

Reply:
Right now in phobos the these values are pretty much the only ones that will work in the storage-project.     I can put in there how to find them but using openstack calls if that would be better but for a quick and dirty guide to test rpc-ceph I thought the values would be enough.    There is only one network that works for us now(but that may change).    There is currently only one image that will work with this hardware flavor

Reply:
thats probably overkill i anticipate ppl can figure that out already :) or should be able to